recipes-tag/netcdf4/.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+Thanks for your interest in helping out conda-forge.
```

Comment:
@licode, do we actually need this directory? Same for `.circleci` above.

Reply:
thanks for correcting. But this repo also needs to be removed as it builds against wrong hdf5 version of 1.10.0. We have new version against on hdf5 1.10.2 in defaults